NiBAx/mainwindow.py

```diff
@@ -75,16 +80,18 @@ def __init__(self, dataFile=None, harmonizationModelFile=None, SPAREModelFile=No
             #if SPARE model file provided on cmd line, load it
             #self.OnSPAREModelFileOpenClicked(SPAREModelFile)
         
-        # Include Mac menu bar
-        self.ui.actionHelp.setMenuRole(QAction.NoRole)
-        self.ui.actionAbout.setMenuRole(QAction.NoRole)
+        ## Include Mac menu bar
+        #self.ui.menuFile.setMenuRole(QAction.NoRole)
+        #self.ui.actionOpen.setMenuRole(QAction.NoRole)
+        #self.ui.actionSave.setMenuRole(QAction.NoRole)
```

Comment:
This is necessary for proper functioning on macOS. This should not be commented out.